solidity/random-beacon/.gitignore

```diff
@@ -0,0 +1,2 @@
+yarn-error.log
```

Comment:
Maybe we could define these properties in root's `.gitignore`?

Reply:
Sure d536174